application/clicommands/RescanCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/* Icinga Web 2 X.509 Module | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\X509\Clicommands;
+
+use Icinga\Application\Logger;
+use Icinga\Module\X509\CertificateUtils;
+
+class RescanCommand extends ScanCommand
+{
+    /**
+     * Rescans already known IP and port to find X.509 certificates.
+     *
+     * USAGE
+     *
+     * icingacli x509 rescan [options]
+     *
+     * OPTIONS:
+     *
+     *   --job=<name>          Rescan known IPs which belong to the given job.
+     *   --last-scan='2days'   Rescan targets whose last scan is older than the specified time
```

Comment:
Rename to `since-last-scan` and don't provide an example directly, but write `--since-last-scan=<datetime>`.

For the description I would write the following:

> Rescans only the targets whose last scan is older than the specified date/time, which can also be an English textual datetime description like "2 days".